identityClient/identityClient.go

```diff
@@ -30,6 +31,53 @@ type E3dbIdentityClient struct {
 	httpClient  *http.Client
 }
 
+// ListRealmApplications lists the applications for a given realm or error (if any).
+func (c *E3dbIdentityClient) ListRealmApplications(ctx context.Context, realmName string) (*ListRealmApplicationsResponse, error) {
+	var applications *ListRealmApplicationsResponse
+	path := c.Host + identityServiceBasePath + "/" + realmResourceName + "/" + realmName + "/" + applicationResourceName
+	request, err := e3dbClients.CreateRequest("GET", path, nil)
+	if err != nil {
+		return applications, err
+	}
+	err = e3dbClients.MakeSignedServiceCall(ctx, request, c.SigningKeys, c.ClientID, &applications)
+	return applications, err
+}
+
+// DeleteRealmApplication deletes the realm with the specified name, returning error (if any).
```

Comment:
Should this say deletes the realm application name?

Reply:
yes! updated